### ISSUE TITLE
Discussion position options not working when using MergeRequestDiscussions.create

### ISSUE THREAD

**Issue by jcroall** [jdalrymple/gitbeaker#2396]
**Description**

- Node.js version: v16.14.0
- Gitbeaker version: 35.4.0
- Gitbeaker release (cli, node, browser, core, requester-utils): node
- OS & version: macOS 11.6

**Expected behaviour**

I was expecting the discussion to show up on the indicated lines! Passing the same values in via PyGitLab works as expected. Looks like this:

![image](https://user-images.githubusercontent.com/1245066/157710958-868dbe34-4ca7-4349-ba6d-20bc5418ed35.png)

**Actual behaviour**

The position is either not sent or not sent properly. I'm not 100% sure how to debug the HTTP request/response to see further.

This is how it looks without the position - not as useful:

![image](https://user-images.githubusercontent.com/1245066/157711174-34e11901-5c7d-46a8-b6b6-0de7ace43dfb.png)

**Possible fixes**

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
Ill give it a look!

**Comment by jdalrymple** [comment]
@jcroall Any chance you could include your code snippet as well?

**Comment by jcroall** [comment]
Sorry, I thought I included some sample code! It looks like there was something broken here, so now perhaps it is user error? This is how I am setting the options:

``
      let opts: BaseRequestOptions = {
        "body": reviewCommentBody,
        "position[position_type]": "text",
        "position[base_sha]": CI_MERGE_REQUEST_DIFF_BASE_SHA,
        "position[start_sha]": CI_MERGE_REQUEST_DIFF_BASE_SHA,
        "position[head_sha]": CI_COMMIT_SHA,
        "position[new_path]": issue.strippedMainEventFilePathname,
        "position[old_path]": issue.strippedMainEventFilePathname,
        "position[new_line]": issue.mainEventLineNumber.toString()
      }
``

And then invoking the call:

``
await api.MergeRequestDiscussions.create(CI_PROJECT_ID, merge_request_iid, reviewCommentBody, opts)
``

A comment is being created, but just a regular comment without the position in the code.

**Comment by jcroall** [comment]
It looks like the patch that was made isn't released - I tried building my own local copy of gitbeaker@master and installing to my project, but with the code above the behavior persists. Beginning to suspect user error but not sure where to go from here!

**Comment by jdalrymple** [comment]
I think youre on the right track and i made an error haha, ill double check things one moment.

**Comment by jcroall** [comment]
Thanks! FWIW this has been a great experience using gitbeaker - I am converting all my Python code to JavaScript because the ecosystem of libraries for DevOps integrations is so much more mature and richer.

**Comment by jdalrymple** [comment]
Im adding some more support right now, but here is the basic way to do it:

```js
services.create(projectId, mergeRequestIId, "the body", {
    position: {
      positionType: "text",
      baseSha: "deec163e3ae602aaa450aa14d66bd10b131f5b37",
      headSha: "53d718d45c231db79dc20908b4d95dbb49805e95",
      startSha: "deec163e3ae602aaa450aa14d66bd10b131f5b37",
      newPath: "README.md",
      oldPath: "README.md",
      newLine: 2
    }
  }
)
```

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`35.6.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/35.6.0) :rocket:
